main.py

```diff
@@ -205,6 +210,9 @@ def main() -> None:
             elif test_result == TestResult.NONZERO_EXIT_ERROR:
                 failed_tests += 1
                 emoji = "💥"
+            elif test_result == TestResult.RUNNER_EXCEPTION:
+                failed_tests += 1
+                emoji = "❗️"
```

Comment:
Oh and how about we swap the emojis with `NONZERO_EXIT_ERROR` here - explosion seems quite fitting for this :D